modules/apps/roles/roles-admin-web/src/main/java/com/liferay/roles/admin/web/internal/display/context/EditRoleAssignmentsManagementToolbarDisplayContext.java

```diff
@@ -419,15 +422,27 @@ public String getSortingURL() {
 		).buildString();
 	}
 
-	public String getTabs2() {
+	public String getTabs2() throws PortalException {
+		long roleId = ParamUtil.getLong(_httpServletRequest, "roleId");
```

Comment:
Please inline roleId like 
```
Role role = RoleServiceUtil.fetchRole(ParamUtil.getLong(_httpServletRequest, "roleId"));
```
because it's used only once and put this line below the first if

Reply:
https://github.com/liferay-commerce/liferay-portal/pull/4499/commits/38caeb68e1dc15adc70bc2759770bb7d520048ea#diff-d2643c71e9b9b6a099e693a45ff57775185c37eae0d640a0ea082731e0148c2dR428